src/Akka.Streams.Kafka/Stages/Consumers/Actors/KafkaConsumerActor.cs

```diff
@@ -391,28 +391,25 @@ private void Poll()
 
             try
             {
-                if (_requests.IsEmpty())
```

Comment:
This optimization is not nescessary in C#, it is already done in the native library and we don't have any control over it.